lsp-jedi.el

```diff
@@ -32,8 +32,7 @@
 (defgroup lsp-jedi nil
   "LSP support for Python, using Jedi Python Language Server."
   :group 'lsp-mode
-  :link '(url-link "https://github.com/fredcamps/lsp-jedi")
-  :package-version '(lsp-mode . "6.0"))
```

Comment:
why did you remove it?

Reply:
Because you used the lsp script to generate this group, and that script is only meant to be used for the lsp project. This is also not a group introduced by lsp-mode, it's a group introduced by lsp-jedi.